code/datums/components/knockoff.dm

```diff
@@ -34,6 +34,10 @@
 		return
 	if(!wearer.dropItemToGround(item))
 		return
+	if(istype(item, /obj/item/toy/basketball))
```

Comment:
this should probably be a callback instead of a hard basketball check

Reply:
Think I did this right? No longer a hard check (i think) but the behavior is still there.